### ISSUE TITLE
Alphabetical order of required paths

### ISSUE THREAD

**Issue by obrejla** [cvisco/eslint-plugin-requirejs#59]
It would be great to have a rule which will check the required paths and will enforce user to have it in alphabetical order.

This will be considered as warning ('foo/baz' should be AFTER 'foo/bar'):

require([
    'foo/baz',
    'foo/bar'
], function (
    baz, 
    bat) {
});

This will NOT be considered as warning ('foo/baz' IS AFTER 'foo/bar'):

require([
    'foo/bat',
    'foo/baz'
], function (
    bat, 
    baz) {
});

Noone cares about order of function params (only path should be checked).

Thanks!

**Comment by cvisco** [comment]
@OndrejBrejla should this be alphabetical based on full path or just or just the basename?
For example, is the following a warning or not?

``` js
require([
    'a/baz',
    'b/bar'
], function (baz, bar) { /* ... */ });
```

**Comment by obrejla** [comment]
This should NOT be warning. It should check alphabetical order based on pull path (or there can be an option for that ;)

**Comment by cvisco** [comment]
I like options :)

**Comment by obrejla** [comment]
I started to work on some implementation, it's almost done (without option ;) so we will see. Once it will be done, I'll push it to my fork.

**Comment by cvisco** [comment]
Looking forward to it!
